component/vshn_postgres.jsonnet

```diff
@@ -883,6 +883,24 @@ local prometheusRule = {
                   },
                 ],
               },
+              {
+                name: 'postgresql-memory',
+                rules: [
+                  {
+                    alert: 'PostgreSQLMemoryCritical',
+                    annotations: {
+                      description: 'The memory claimed by {{ $labels.pod }} has been over 85% for 2 hours.\n  Please reducde the load of this instance, or increase the memory.',
+                      runbook_url: 'TBD',
```

Comment:
Nitpick: If we don't have a runbook I wouldn't add `TBD`. It really doesn't help us remembering this and only looks bad if a customer sees it